cmd/armadactl/README.md

```diff
@@ -0,0 +1,105 @@
+# Armadactl
+
+Armadactl is a command line tool for managing and deploying Armada charts. It allows users to install, upgrade, and delete charts, as well as view their status and history.
```

Comment:
Not sure what you mean here.  Armadactl is for interacting with armada objects via the cli ie creating queues, submitting jobs, watching events.  NOt sure what you mean by charts or status or history.